src/main/service/listParseHelperService.ts

```diff
@@ -0,0 +1,359 @@
+    let caseHearingChannel = '';
+    if (sitting['channel'] || session['sessionChannel']) {
+      if (sitting['channel']?.length > 0) {
+        caseHearingChannel = sitting['channel'].join(', ');
+      } else if (session['sessionChannel'].length > 0) {
+        caseHearingChannel = session['sessionChannel'].join(', ');
+      }
+    }
+    sitting['caseHearingChannel'] = caseHearingChannel;
+  }
+
+  /**
+   * Manipulate judiciary data for writing out to screen.
+   * @param session
+   */
+  public findAndManipulateJudiciary(session: object): string {
+    let judiciaries = '';
+    let foundPresiding = false;
+    session['judiciary']?.forEach(judiciary => {
+      if (judiciary?.isPresiding === true) {
+        judiciaries = ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs);
+        foundPresiding = true;
+      } else if (!foundPresiding) {
+        if (ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs) !== '') {
+          judiciaries += ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs) + ', ';
+        }
+      }
+    });
+
+    if (!foundPresiding) {
+      judiciaries = judiciaries.slice(0, -2);
+    }
+
+    return judiciaries;
+  }
+
+  /**
+   * Format linked cases by joining individual case ID with delimiter
+   * @param hearing
+   */
+  public findAndManipulateLinkedCases(hearing: object): void {
+    hearing['case'].forEach(hearingCase => {
+      let linkedCases = '';
+      let counter = 1;
+      hearingCase['caseLinked']?.forEach(linkedCase => {
+        linkedCases += (counter == hearingCase['caseLinked'].length)
+          ? linkedCase['caseId']
+          : linkedCase['caseId'] + ', ';
+        counter++;
+      });
+      hearingCase['formattedLinkedCases'] = linkedCases;
+    });
+  }
+
+  /**
+   * Calculate the duration of a sitting.
+   * @param sitting
+   */
+  public calculateDuration(sitting: object): void {
+    sitting['duration'] = '';
+    sitting['startTime'] = '';
+    if (sitting['sittingStart'] !== '' && sitting['sittingEnd'] !== '') {
+      const sittingStart = moment.utc(sitting['sittingStart']);
+      const sittingEnd = moment.utc(sitting['sittingEnd']);
+      let durationAsHours = 0;
+      let durationAsMinutes = moment.duration(sittingEnd.startOf('minutes').diff(sittingStart.startOf('minutes'))).asMinutes();
+      if (durationAsMinutes >= 60) {
+        durationAsHours = Math.floor(durationAsMinutes / 60);
+        durationAsMinutes = durationAsMinutes - (durationAsHours * 60);
+      }
+
+      let durationAsDays = 0;
+      if(durationAsHours >= 24) {
+        durationAsDays = Math.floor(durationAsHours / 24);
+      }
+
+      sitting['durationAsHours'] = durationAsHours;
+      sitting['durationAsMinutes'] = durationAsMinutes;
+      sitting['durationAsDays'] = durationAsDays;
+
+      sitting['time'] = moment.utc(sitting['sittingStart']).tz(this.timeZone).format('HH:mm');
```

Comment:
I wonder line 276 and 282 should be moved out to a different method. It has got nothing to do with duration and shoehorned into this method. Also some of the lists using 'time', and some lists using 'startTime' but all for the same thing only different format. If you do that will need to find out what lists using these 2 fields and they need to call this method individually. So this could be a lot more work for u. We can log it and fix this issue separately so it is up to u.